app/api/discount_codes.py

```diff
@@ -256,7 +254,7 @@ def before_get(self, args, kwargs):
             kwargs['discount_event_id'] = event.id
 
         if kwargs.get('event_id') and has_access('is_admin'):
-            event = safe_query(db, Event, 'id', kwargs['event_id'], 'event_id')
+            event = safe_query(Event, 'id', kwargs['event_id'], 'event_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)